src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -189,8 +185,8 @@ const BlastSettings = ({ config }: Props) => {
           </div>
           <div>
             <BlastSelect
-              options={getPresetsList(config).options}
-              label={getPresetsList(config).label}
+              options={getPresetsList(config, blastProgram).options}
+              label={getPresetsList(config, blastProgram).label}
```

Comment:
let's create a `presetsList` variable before the return, instead of calling the `getPresetsList` function twice

Reply:
✅